recipes/langsmith/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "langsmith" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langsmith-{{ version }}.tar.gz
+  sha256: cf317b47c506f8c89a3b3a932d6fbcfb45ae1f43ed555d23b71d166ab9db21cf
```

Comment:
looks like when you updated the version, you didn't update the sha256.

Reply:
Got it! I'll update. Thank you for your help :)